app/templates/base.html

```diff
@@ -63,6 +88,7 @@ <h3 class="h3">Song Recommender</h3>
             <a href="{{ url_for('logout') }}" class="btn btn-link">Logout</a>
         </section>
     </header>
+>>>>>>> 496051eca505e2a2c986b02c9219ed622a606e09
```

Comment:
This merge conflict should be resolved before merging to `main`. To do this, delete this line (and similar lines starting with `>>>>` or `<<<<<` according to the changes you want to keep) then commit these changes. Once you commit, those changes should be reflected as part of this PR